packages/transaction-controller/src/gas-flows/OracleLayer1GasFeeFlow.ts

```diff
@@ -0,0 +1,122 @@
+import { Common, Hardfork } from '@ethereumjs/common';
+import { TransactionFactory } from '@ethereumjs/tx';
+import { Contract } from '@ethersproject/contracts';
+import { Web3Provider, type ExternalProvider } from '@ethersproject/providers';
+import type { Hex } from '@metamask/utils';
+import { createModuleLogger } from '@metamask/utils';
+import { omit } from 'lodash';
+
+import { projectLogger } from '../logger';
+import type {
+  Layer1GasFeeFlow,
+  Layer1GasFeeFlowRequest,
+  Layer1GasFeeFlowResponse,
+  TransactionMeta,
+} from '../types';
+
+const log = createModuleLogger(projectLogger, 'optimisim-layer1-gas-fee-flow');
```

Comment:
It seems this should be named `oracle-layer1-gas-fee-flow`, unless we decide against initializing a logger in the constructor for instances that extend this class.

Reply:
Good spot! Done.